examples/lock-app/efr32/include/LockManager.h

```diff
@@ -31,15 +31,26 @@
 
 using namespace ::chip;
 
-// Currently up to 10 users are support on the EFR32 platform
+// up to 10 users are supported on the EFR32 platform
 #define DOOR_LOCK_MAX_USERS 10
+// max credential size supported is 8
 #define DOOR_LOCK_MAX_CREDENTIAL_SIZE 8
-#define MININUM_USER_INDEX 1
-#define MINIMUM_CREDENTIAL_INDEX 1
+// up to 10 schedules of each type supported
+#define DOOR_LOCK_MAX_SCHEDULES 10
```

Comment:
I realize this is existing pattern/code however I believe some `static constexpr uint8_t kDoorLockMaxSchedules = 10` would be more typesafe. Or even better, namespace to make it clear that these are for locks (why is max schedules named with a DOOR_LOCK prefix but not schedules per user?)

```cpp

namespace DoorLock {
namespace ResourceRanges {
   static constexpr uint8_t kMaxCredentialSize = 8;
   static constexpr uint8_t kMaxSchedules = 10;
   static constexpr uint8_t kMaxCredentialsPerUser = 10;
   static constexpr uint8_t kMaxSchedulesPerUser = 10;

   // Indices received for user/credential/schedules are 1-indexed
   static constexpr uint8_t kStartIndexValue = 1;
   
   // Assume up to 10 users
   static constexpr uint8_t kMaxCredentials = 10 * kMaxCredentialsPerUser;
} // namespace ResourceRanges
} // namespace DoorLock
```

Reply:
I like this. I read up on this, and this method seems preferred over preprocessor directives. Using this with an EFR32DoorLock namespace (as DoorLock is already used).